queue_services/entity-filer/src/entity_filer/filing_processors/annual_report.py

```diff
@@ -44,6 +45,13 @@ def process(business: Business, filing: Dict, filing_meta: FilingMeta):
 
     business.last_ar_year = business.last_ar_year + 1 if business.last_ar_year else business.founding_date.year + 1
 
+    # remove dissolution flag if business can be withdrawn
+    if flag_on and business.in_dissolution:
+        eligibility, _ = InvoluntaryDissolutionService.check_business_eligibility(business.identifier, False)
+        if not eligibility:
+            batch_processing, _ = InvoluntaryDissolutionService.get_in_dissolution_batch_processing(business.id)
+            batch_processing.status = BatchProcessing.BatchProcessingStatus.WITHDRAWN.value
```

Comment:
Can you also update the following fields:
* `batch_processing.notes` = "Moved back to good standing"
* `batch_processing.last_modified` = now

Reply:
Okay, I will